Adds SparkFun Blynk Board

edspark opened the pull request:
I have a sequence of four commits that hopefully clarifies the steps I took to add the SparkFun Blynk board. I ran the python script (board.txt.py) as instructed and have included all files generated including the boards.txt.orig in this pull request. Let me know if I missed anything and I'll make sure to change/modify whatever it is as soon as possible. Thanks!

d-a-v commented:
Can you please:
- make `tools/boards.txt.py` executable again (you changed its mode 0755->0644)
- remove `boards.txt.orig` from your PR
- remove `doc/boards.rst.orig`
- remove `package/package_esp8266com_index.template.json.orig`

edspark commented:
I removed the generated files, but was a bit confused as to how I changed permissions on the python script (I'm on a Windows machine). I did try to change it using git bash but we'll see if that worked. Also: I moved the file back to the /tools directory, I had left it in the main directory.

d-a-v commented:
See https://github.com/sparkfun/Arduino/pull/1#issuecomment-549599657
